### PR TITLE
Readme Updated

### PR DESCRIPTION
**Pull request by revanthkumarJ**
## Issue Fix
Fixes #{Issue Number}
Jira Task: [Task_Number](Task_Link)

## Description
Changed sentences to a formal way

##

- [ ] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [ ] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [ ] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by revanthkumarJ** [comment]
@niyajali I have changed the things you commented previously. Please review it once

File: README.md [hunk 1]
```diff
@@ -61,82 +66,64 @@ that can be used as a dependency in any other wallet based project. It is develo
...
+Occasionally, merge conflicts may arise when your pull request is being reviewed. These conflicts need to be resolved manually.  
+To learn how to resolve merge conflicts, please refer to the [Solving Merge Conflicts](https://github.com/openMF/mobile-wallet/wiki/Solving-Merge-Conflicts) guide.
 
 ### Conclusion
 By following these contribution guidelines, you're all set to start contributing to the Mobile Wallet (Mifos Pay) project. We appreciate your efforts and look forward to your valuable contributions. Happy coding!
 
 ## Instructions to get the latest APK
-
 To get the latest apk fom the Github actions artifacts, follow these steps:
-
```
Comment (niyajali) [inline README.md:132]: https://github.com/openMF/mobile-wallet/actions/workflows/master_dev_ci.yml
use this link to download artifacts for all platform and mention steps to download application for different platform.

File: README.md [hunk 2]
```diff
@@ -1,19 +1,24 @@
+![Kotlin](https://img.shields.io/badge/Kotlin-7f52ff?style=flat-square&logo=kotlin&logoColor=white) 
+![Kotlin Multiplatform](https://img.shields.io/badge/Kotlin%20Multiplatform-4c8d3f?style=flat-square&logo=kotlin&logoColor=white)
+![Compose Multiplatform](https://img.shields.io/badge/Jetpack%20Compose%20Multiplatform-000000?style=flat-square&logo=android&logoColor=white)
+
```
Comment (niyajali) [inline README.md:6]: [![Mobile-Wallet CI[Master/Dev]](https://github.com/openMF/mobile-wallet/actions/workflows/master_dev_ci.yml/badge.svg?branch=dev)](https://github.com/openMF/mobile-wallet/actions/workflows/master_dev_ci.yml) add this badge

File: README.md [hunk 3]
```diff
@@ -61,82 +66,64 @@ that can be used as a dependency in any other wallet based project. It is develo
...
+The project fully utilizes [Jetpack Compose](https://developer.android.com/jetpack/compose) with complete support for **Kotlin Multiplatform**.
 
-Contributions are welcome in the following areas:
-- Development and enhancements related to Kotlin Multiplatform.
-- Logical changes or improvements in existing Jetpack Compose-based features.
+We invite contributions in the following areas:
+- Development of new features and enhancements using Kotlin Multiplatform.
+- Improvements and refinements to existing Jetpack Compose-based functionalities.
 
-We appreciate your contributions and look forward to collaborating with you!
```
Comment (niyajali) [inline README.md:72]: keep this line which is better than below line

File: README.md [hunk 4]
```diff
@@ -61,82 +66,64 @@ that can be used as a dependency in any other wallet based project. It is develo
...
 |--------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------|
-| ![Mobile-Wallet CI[Master/Dev]](https://github.com/openMF/mobile-wallet/workflows/Mobile-Wallet%20CI%5BMaster/Dev%5D/badge.svg?branch=dev) | [![Join the chat at https://mifos.slack.com/](https://img.shields.io/badge/Join%20Our%20Community-Slack-blue)](https://mifos.slack.com/) |
+| [![Mobile-Wallet CI[Master/Dev]](https://github.com/openMF/mobile-wallet/workflows/Mobile-Wallet%20CI%5BMaster/Dev%5D/badge.svg?branch=dev)](https://github.com/openMF/mobile-wallet/actions/workflows/master_dev_ci.yml) | [![Join the chat at https://mifos.slack.com/](https://img.shields.io/badge/Join%20Our%20Community-Slack-blue)](https://mifos.slack.com/) |
```
Comment (niyajali) [inline README.md]: moves this badges to top with other badges

**Review by niyajali — CHANGES_REQUESTED** [review]

**Comment by revanthkumarJ** [comment]
@niyajali I have done changes to home readme based on your suggestions and created installation guide readmes for android,linux,windows along with images.
Have a loot at it once

File: README.md [hunk 5]
```diff
@@ -61,82 +69,74 @@ that can be used as a dependency in any other wallet based project. It is develo
...
-## Wiki
+2. Click the latest workflow from the workflows list.  
+
+3. Scroll down to the **Artifacts** section, where you will find: **Android APKs** | **Linux-App** | **MacOS-App** | **Windows-Apps**. Download all the available zip files.  
+  <img height='200' src="https://github.com/user-attachments/assets/ab7783ff-3834-4be2-9ce8-6342746b22a2">
 
+  
+
+5. After successful download, extract the zip files to your preferred location.  
+
```
Comment (niyajali) [inline README.md:140]: We do not need an installation guide, people know how to install the application, remove those guides

**Review by niyajali — CHANGES_REQUESTED** [review]

File: README.md (hunk 5 above)
Comment (revanthkumarJ) [reply README.md:140]: Removed Installation Guides. Is everything ok now

**Comment by niyajali** [comment]
@revanthkumarJ Okay! It looks good to me. It can be merged, and the screenshot table of this readme can be updated to be full-width and responsive.

**Comment by revanthkumarJ** [comment]
@niyajali Added Responsiveness to screenshot table and squashed commits to a single commit

**Pull request merged**